api/jira/report_qa_requests.py

```diff
@@ -0,0 +1,200 @@
+import inspect
+import pandas as pd
+
+from database import (
+    Database,
+    ReportJiraQARequests,
+    ReportJIraQARequestsNewIssueType,
+)
+
+from api.jira.client import Jira
+from api.jira.helpers import jira_delete
+from utils.datetime_utils import DatetimeUtils as dt
+
+
+_DB = None
+_JIRA = None
+
+
+def _db() -> Database():
+    global _DB
+    if _DB is None:
+        _DB = Database()
+    return _DB
+
+
+def _jira() -> Jira():
+    global _JIRA
+    if _JIRA is None:
+        _JIRA = Jira()
+    return _JIRA
+
+# ===================================================================
+# ORCHESTRATOR (BATCH)
+# ===================================================================
+
+
+def jira_qa_requests():
+    jira = _jira()
+    payload = jira.filters()
+
+    jira_delete(ReportJiraQARequests)
+
+    # Normalize the JSON data
+    df = pd.json_normalize(payload, sep='_')
+
+    # Ensure fields_labels exists
+    if 'fields_labels' not in df.columns:
+        df['fields_labels'] = [[] for _ in range(len(df))]
+
+    # Check if 'jira_assignee_username' exists
+    # if not use 'alternative_assignee_emailAddress'
+    if 'fields_assignee_emailAddress' not in df.columns:
+        df['fields_assignee_emailAddress'] = df.get('fields_assignee', "None") # noqa
+    else:
+        df['fields_assignee_emailAddress'] = df['fields_assignee_emailAddress'].fillna("Not Assigned") # noqa
+
+    # Drop the alternative column if it exists
+    if 'fields_assignee' in df.columns:
+        df.drop(columns=['fields_assignee'], inplace=True)
+
+    # Select specific columns
+    selected_columns = {
+        'key': 'jira_key',
+        'fields_summary': 'jira_summary',
+        'fields_created': 'jira_created_at',
+        'fields_customfield_10155_value': 'jira_firefox_release_train',
+        'fields_customfield_10134_value': 'jira_engineering_team',
+        'fields_customfield_10037': 'jira_story_points',
+        'fields_status_name': 'jira_status',
+        'fields_assignee_emailAddress': 'jira_assignee_username',
+        'fields_labels': 'jira_labels'
+    }
+
+    # Select specific columns
+    df_selected = df[selected_columns.keys()]
```

Comment:
This will raise `KeyError` if expected columns are missing from the Jira response. Suggest Try/Except wrapping